tests/config.json

```diff
@@ -31,15 +31,21 @@
         "replication": {
             "topic": "backbeat-test-replication",
             "replicationStatusTopic": "backbeat-test-replication-status",
-            "groupId": "backbeat-test-replication-group"
+            "groupId": "backbeat-test-replication-group",
+            "destination": {
```

Comment:
Maybe you didn't intend to commit this? Though in practice should not be an issue.

Reply:
ah yes, I think I was getting ahead of myself here, these changes are already reflected for PR#193 and I think I included this here without much thought, sorry about that